src/components/Navigation/Navigation.tsx

```
@@ -1,8 +1,9 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
 import './Navigation.css';
+import { useTranslation } from 'react-i18next';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useTranslation'. Consider to delete it or rename it.

'useTranslation' is defined here:


(lint/suspicious/noRedeclare)